anax-in-container/horizon-container

```diff
@@ -378,6 +378,7 @@ stop() {
 
 	# Stop the anax container
 	echo "Unregistering the node, then stopping/removing the horizon container (this may take a minute)..."
+	/usr/bin/hzn unregister -f
```

Comment:
Before you issue the unregister command, I think you need to issue
```
export HORIZON_URL=http://localhost:$HORIZON_AGENT_PORT
```
to make sure you are talking to the right container... I could have 2 or 3 anax-in-containers running and say I want to stop horizon2... This will get it to the correct one